tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/TypeSpec/TypeSpecProjectInPublicRepo.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.ComponentModel;
+using Azure.Sdk.Tools.Cli.Contract;
+using Azure.Sdk.Tools.Cli.Helpers;
+using ModelContextProtocol.Server;
+
+namespace Azure.Sdk.Tools.Cli.Tools
+{
+    /// <summary>
+    /// This tool is used to check if a TypeSpec project is in a public spec repository.
+    /// </summary>
+    [Description("TypeSpec public repository validation tool")]
+    [McpServerToolType]
+    public class TypeSpecProjectInPublicRepo(ITypeSpecHelper typeSpecHelper, ILogger<TypeSpecProjectInPublicRepo> logger) : MCPTool
```

Comment:
Please rename this so it ends with `Tool`. One suggestion is TypeSpecPublicRepoValidationTool